southworkscom/botframework-solutions
# [TypeScript][Botbuilder Solutions/Botbuilder Skill lib] Remove unused dependencies

## Description 

Removed unused dependencies from `package.json`in the following project:
  - Botbuilder Solutions
  - Botbuilder Skills

## Testing Steps 
1. Go to  `.\botframework-solutions\lib\typescript`
1. Open the console
1. Execute `rush install` to install dependencies
1. Run the build `rush build`
1. Run the test `rush test`
1. Verify that the test run successfully
